### PR TITLE
Move labels webhooks to notification

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: services/issue/label.go [hunk 1]
```diff
@@ -84,7 +36,7 @@ func AddLabels(issue *models.Issue, doer *models.User, labels []*models.Label) e
 		return err
 	}
 
-	sendLabelUpdatedWebhook(issue, doer)
+	notification.NotifyIssueChangeLabels(doer, issue, nil, nil)
```
Comment (6543) [inline services/issue/label.go]:
```suggestion
	notification.NotifyIssueChangeLabels(doer, issue, labels, nil)
```
why not use the new parameter - we can implement the handling at NotifyIssueChangeLabels later?

File: services/issue/label.go [hunk 2]
```diff
@@ -74,7 +26,7 @@ func AddLabel(issue *models.Issue, doer *models.User, label *models.Label) error
...
```
Comment (6543) [inline services/issue/label.go]:
```suggestion
	notification.NotifyIssueChangeLabels(doer, issue, []*models.lable {label}, nil)
```
same here ...

File: services/issue/label.go [hunk 3]
```diff
@@ -106,6 +58,6 @@ func RemoveLabel(issue *models.Issue, doer *models.User, label *models.Label) er
...
```
Comment (6543) [inline services/issue/label.go]:
```suggestion
	notification.NotifyIssueChangeLabels(doer, issue, nil, []*models.lable {label})
```
same here ...

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@lunny optional changes aboth ... beside that approved :D

File: modules/notification/webhook/webhook.go [hunk 4]
```diff
@@ -398,3 +398,50 @@ func (m *webhookNotifier) NotifyDeleteComment(doer *models.User, comment *models
 		go models.HookQueue.Add(comment.Issue.Repo.ID)
 	}
 }
+
+func (m *webhookNotifier) NotifyIssueChangeLabels(doer *models.User, issue *models.Issue,
```
Comment (guillep2k) [inline modules/notification/webhook/webhook.go:403]: I see that `ClearLabels()` doesn't use this function. Is it a whole type of notification?

**Comment by lunny** [comment]
@6543 all done.
@guillep2k ClearLabels invoked `NotifyIssueClearLabels` but not `NotifyIssueChangeLabels`.

**Review by guillep2k — APPROVED** [review]

**Pull request merged**